packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -712,17 +747,27 @@ class _AndroidPlatformView extends LeafRenderObjectWidget {
   }
 }
 
-class _UiKitPlatformView extends LeafRenderObjectWidget {
-  const _UiKitPlatformView({
+abstract class _DarwinPlatformView<TController extends DarwinPlatformViewController, TRender extends RenderDarwinPlatformView<TController>> extends LeafRenderObjectWidget {
+  const _DarwinPlatformView({
     required this.controller,
     required this.hitTestBehavior,
     required this.gestureRecognizers,
   });
 
-  final UiKitViewController controller;
+  final TController controller;
   final PlatformViewHitTestBehavior hitTestBehavior;
   final Set<Factory<OneSequenceGestureRecognizer>> gestureRecognizers;
 
+  @override
+  void updateRenderObject(BuildContext context, TRender renderObject) {
+    renderObject.viewController = controller;
+    renderObject.hitTestBehavior = hitTestBehavior;
```

Comment:
Convention is to use cascades here:

```suggestion
    renderObject
      ..viewController = controller
      ..hitTestBehavior = hitTestBehavior;
```

Reply:
Do you think changes such as this to the code that was left unchanged aside from renaming the enclosing class, such as this, should be within the scope of this PR?

Reply:
For me, tiny clean-ups like this are always in scope, especially if you touch the code anyways like here.